### ISSUE TITLE
UnicodeEncodeError: 'latin-1' codec can't encode characters in position 7120-7122: ordinal not in range(256)

### ISSUE THREAD

**Issue by stdedos** [Xpra-org/xpra#4198]
**Describe the bug**

```python
2024-04-24 22:46:42,417 xpra main error:
Traceback (most recent call last):
  File "E:/xpra/xpra/scripts/main.py", line 2273, in run_remote_server
  File "E:/xpra/xpra/scripts/main.py", line 1026, in connect_or_fail
  File "E:/xpra/xpra/scripts/main.py", line 1131, in connect_to
  File "E:/xpra/xpra/net/ssh/exec_client.py", line 128, in connect_to
xpra.scripts.config.InitExit: Error running ssh command 'plink -ssh -agent -l u -T h \'sh -c \'"\'"\'if command -v "xpra" > /dev/null 2>&1; then xpra _proxy_shadow_start ":1" "--opengl=no" "--title=@title@ on @@/@server-display@" "--ssh=plink -ssh -agent" "--microphone=off" "--speaker=off" "--webcam=no" "--headerbar=off" "--min-speed=70" "--sync-xvfb=False" "--pulseaudio=no" "--modal-windows=no" "--encodings=h264,vp9,vp8,mpeg4,mpeg4+mp4,h264+mp4,vp8+webm,vp9+webm,png,png/P,png/L,avif,rgb,rgb24,rgb32,jpeg,jpega,h265,av1,scroll,grayscale,stream" "--key-shortcut=Control+Menu:toggle_keyboard_grab" "--key-shortcut=Shift+Menu:toggle_pointer_grab" "--key-shortcut=Shift+F11:toggle_fullscreen" "--key-shortcut=#+F1:show_menu" "--key-shortcut=#+F2:show_start_new_command" "--key-shortcut=#+F3:show_bug_report" "--key-shortcut=#+F4:quit" "--key-shortcut=#+F5:show_window_info" "--key-shortcut=#+F6:show_shortcuts" "--key-shortcut=#+F7:show_docs" "--key-shortcut=#+F8:toggle_keyboard_grab" "--key-shortcut=#+F9:toggle_pointer_grab" "--key-shortcut=#+F10:magic_key" "--key-shortcut=#+F11:show_session_info" "--key-shortcut=#+F12:toggle_debug" "--key-shortcut=#+plus:scaleup" "--key-shortcut=#+minus:scaledown" "--key-shortcut=#+underscore:scaledown" "--key-shortcut=#+KP_Add:scaleup" "--key-shortcut=#+KP_Subtract:scaledown" "--key-shortcut=#+KP_Multiply:scalereset" "--key-shortcut=#+bar:scalereset" "--key-shortcut=#+question:scalingoff" "--key-shortcut=Shift+F2:toggle_pointer_grab";elif [ -x $XDG_RUNTIME_DIR/xpra/run-xpra ]; then $XDG_RUNTIME_DIR/xpra/run-xpra _proxy_shadow_start ":1" "--opengl=no" "--title=@title@ on @@/@server-display@" "--ssh=plink -ssh -agent" "--microphone=off" "--speaker=off" "--webcam=no" "--headerbar=off" "--min-speed=70" "--sync-xvfb=False" "--pulseaudio=no" "--modal-windows=no" "--encodings=h264,vp9,vp8,mpeg4,mpeg4+mp4,h264+mp4,vp8+webm,vp9+webm,png,png/P,png/L,avif,rgb,rgb24,rgb32,jpeg,jpega,h265,av1,scroll,grayscale,stream" "--key-shortcut=Control+Menu:toggle_keyboard_grab" "--key-shortcut=Shift+Menu:toggle_pointer_grab" "--key-shortcut=Shift+F11:toggle_fullscreen" "--key-shortcut=#+F1:show_menu" "--key-shortcut=#+F2:show_start_new_command" "--key-shortcut=#+F3:show_bug_report" "--key-shortcut=#+F4:quit" "--key-shortcut=#+F5:show_window_info" "--key-shortcut=#+F6:show_shortcuts" "--key-shortcut=#+F7:show_docs" "--key-shortcut=#+F8:toggle_keyboard_grab" "--key-shortcut=#+F9:toggle_pointer_grab" "--key-shortcut=#+F10:magic_key" "--key-shortcut=#+F11:show_session_info" "--key-shortcut=#+F12:toggle_debug" "--key-shortcut=#+plus:scaleup" "--key-shortcut=#+minus:scaledown" "--key-shortcut=#+underscore:scaledown" "--key-shortcut=#+KP_Add:scaleup" "--key-shortcut=#+KP_Subtract:scaledown" "--key-shortcut=#+KP_Multiply:scalereset" "--key-shortcut=#+bar:scalereset" "--key-shortcut=#+question:scalingoff" "--key-shortcut=Shift+F2:toggle_pointer_grab";elif [ -x /usr/local/bin/xpra ]; then /usr/local/bin/xpra _proxy_shadow_start ":1" "--opengl=no" "--title=@title@ on @@/@server-display@" "--ssh=plink -ssh -agent" "--microphone=off" "--speaker=off" "--webcam=no" "--headerbar=off" "--min-speed=70" "--sync-xvfb=False" "--pulseaudio=no" "--modal-windows=no" "--encodings=h264,vp9,vp8,mpeg4,mpeg4+mp4,h264+mp4,vp8+webm,vp9+webm,png,png/P,png/L,avif,rgb,rgb24,rgb32,jpeg,jpega,h265,av1,scroll,grayscale,stream" "--key-shortcut=Control+Menu:toggle_keyboard_grab" "--key-shortcut=Shift+Menu:toggle_pointer_grab" "--key-shortcut=Shift+F11:toggle_fullscreen" "--key-shortcut=#+F1:show_menu" "--key-shortcut=#+F2:show_start_new_command" "--key-shortcut=#+F3:show_bug_report" "--key-shortcut=#+F4:quit" "--key-shortcut=#+F5:show_window_info" "--key-shortcut=#+F6:show_shortcuts" "--key-shortcut=#+F7:show_docs" "--key-shortcut=#+F8:toggle_keyboard_grab" "--key-shortcut=#+F9:toggle_pointer_grab" "--key-shortcut=#+F10:magic_key" "--key-shortcut=#+F11:show_session_info" "--key-shortcut=#+F12:toggle_debug" "--key-shortcut=#+plus:scaleup" "--key-shortcut=#+minus:scaledown" "--key-shortcut=#+underscore:scaledown" "--key-shortcut=#+KP_Add:scaleup" "--key-shortcut=#+KP_Subtract:scaledown" "--key-shortcut=#+KP_Multiply:scalereset" "--key-shortcut=#+bar:scalereset" "--key-shortcut=#+question:scalingoff" "--key-shortcut=Shift+F2:toggle_pointer_grab";elif [ -x ~/.xpra/run-xpra ]; then ~/.xpra/run-xpra _proxy_shadow_start ":1" "--opengl=no" "--title=@title@ on @@/@server-display@" "--ssh=plink -ssh -agent" "--microphone=off" "--speaker=off" "--webcam=no" "--headerbar=off" "--min-speed=70" "--sync-xvfb=False" "--pulseaudio=no" "--modal-windows=no" "--encodings=h264,vp9,vp8,mpeg4,mpeg4+mp4,h264+mp4,vp8+webm,vp9+webm,png,png/P,png/L,avif,rgb,rgb24,rgb32,jpeg,jpega,h265,av1,scroll,grayscale,stream" "--key-shortcut=Control+Menu:toggle_keyboard_grab" "--key-shortcut=Shift+Menu:toggle_pointer_grab" "--key-shortcut=Shift+F11:toggle_fullscreen" "--key-shortcut=#+F1:show_menu" "--key-shortcut=#+F2:show_start_new_command" "--key-shortcut=#+F3:show_bug_report" "--key-shortcut=#+F4:quit" "--key-shortcut=#+F5:show_window_info" "--key-shortcut=#+F6:show_shortcuts" "--key-shortcut=#+F7:show_docs" "--key-shortcut=#+F8:toggle_keyboard_grab" "--key-shortcut=#+F9:toggle_pointer_grab" "--key-shortcut=#+F10:magic_key" "--key-shortcut=#+F11:show_session_info" "--key-shortcut=#+F12:toggle_debug" "--key-shortcut=#+plus:scaleup" "--key-shortcut=#+minus:scaledown" "--key-shortcut=#+underscore:scaledown" "--key-shortcut=#+KP_Add:scaleup" "--key-shortcut=#+KP_Subtract:scaledown" "--key-shortcut=#+KP_Multiply:scalereset" "--key-shortcut=#+bar:scalereset" "--key-shortcut=#+question:scalingoff" "--key-shortcut=Shift+F2:toggle_pointer_grab";elif [ -x Xpra_cmd.exe ]; then Xpra_cmd.exe _proxy_shadow_start ":1" "--opengl=no" "--title=@title@ on @@/@server-display@" "--ssh=plink -ssh -agent" "--microphone=off" "--speaker=off" "--webcam=no" "--headerbar=off" "--min-speed=70" "--sync-xvfb=False" "--pulseaudio=no" "--modal-windows=no" "--encodings=h264,vp9,vp8,mpeg4,mpeg4+mp4,h264+mp4,vp8+webm,vp9+webm,png,png/P,png/L,avif,rgb,rgb24,rgb32,jpeg,jpega,h265,av1,scroll,grayscale,stream" "--key-shortcut=Control+Menu:toggle_keyboard_grab" "--key-shortcut=Shift+Menu:toggle_pointer_grab" "--key-shortcut=Shift+F11:toggle_fullscreen" "--key-shortcut=#+F1:show_menu" "--key-shortcut=#+F2:show_start_new_command" "--key-shortcut=#+F3:show_bug_report" "--key-shortcut=#+F4:quit" "--key-shortcut=#+F5:show_window_info" "--key-shortcut=#+F6:show_shortcuts" "--key-shortcut=#+F7:show_docs" "--key-shortcut=#+F8:toggle_keyboard_grab" "--key-shortcut=#+F9:toggle_pointer_grab" "--key-shortcut=#+F10:magic_key" "--key-shortcut=#+F11:show_session_info" "--key-shortcut=#+F12:toggle_debug" "--key-shortcut=#+plus:scaleup" "--key-shortcut=#+minus:scaledown" "--key-shortcut=#+underscore:scaledown" "--key-shortcut=#+KP_Add:scaleup" "--key-shortcut=#+KP_Subtract:scaledown" "--key-shortcut=#+KP_Multiply:scalereset" "--key-shortcut=#+bar:scalereset" "--key-shortcut=#+question:scalingoff" "--key-shortcut=Shift+F2:toggle_pointer_grab";else echo "no run-xpra command found"; exit 1; fi\'"\'"\'\'': [WinError 2] Δεν είναι δυνατή η εύρεση του καθορισμένου αρχείου από το σύστημα

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:/xpra/xpra/scripts/main.py", line 138, in main
  File "E:/xpra/xpra/scripts/main.py", line 516, in run_mode
  File "E:/xpra/xpra/scripts/main.py", line 574, in do_run_mode
  File "E:/xpra/xpra/scripts/main.py", line 2287, in run_remote_server
  File "E:/xpra/xpra/client/base/client.py", line 177, in show_progress
UnicodeEncodeError: 'latin-1' codec can't encode characters in position 7120-7122: ordinal not in range(256)


"was: Xpra-Light-x86_64_6.0-r35630\xpra_cmd" shadow ssh://user@ip/1 --ssh="plink -ssh -agent" --modal-windows=no --headerbar=off --title="@title@ on @@/@server-display@" --encodings=-webp --microphone=off --speaker=off --webcam=no --pulseaudio=no --opengl=no --min-speed=70 --key-shortcut=Shift+F2:toggle_pointer_grab
```

**To Reproduce**
1. server command
2. client command
3. specific action to trigger the bug

**System Information (please complete the following information):**

- Server OS: Ubuntu 20.04.6 LTS
- Client OS: Microsoft Windows [Version 10.0.18363.836]
- Xpra Server Version: v5.0.8-r0
- Xpra Client Version: (see above)

**Additional context**
Please see "reporting bugs" in the wiki section.

**Comment by totaam** [comment]
> `[WinError 2] Δεν είναι δυνατή η εύρεση του καθορισμένου αρχείου από το σύστημα`

Google translate says that this means: `[WinError 2] The system cannot find the specified file`

> `UnicodeEncodeError: 'latin-1' codec can't encode characters in position 7120-7122: ordinal not in range(256)`

Should be fixed in the commit above. Feel free to re-open if not.

**Comment by stdedos** [comment]
The first issue is still broken in Light-r35715

```python
"Xpra-Light-x86_64_6.1-r35715\xpra_cmd" shadow ssh://user@ip/1 --ssh="plink -ssh -agent" --modal-windows=no --headerbar=off --title="@title@ on @@/@server-display@" --encodings=-webp --microphone=off --speaker=off --webcam=no --pulseaudio=no --opengl=no --min-speed=70 --key-shortcut=Shift+F2:toggle_pointer_grab

XPRA_EXECUTABLE=Xpra-Light-x86_64_6.1-r35715


2024-05-04 12:58:57,403 Xpra GTK3 client version 6.1-r35715 (g1ec038bf9a) beta (light build)
2024-05-04 12:58:57,697  running on Microsoft Windows Microsoft Windows 10 Enterprise
2024-05-04 12:58:57,697  cpython 3.11
2024-05-04 12:58:58,523 GStreamer version 1.24.3
2024-05-04 12:58:58,547 created named pipe 'Xpra\32920'
2024-05-04 12:58:58,681 keyboard layout 'United States - English' : 'us' (0x409)
2024-05-04 12:58:58,906 Error running ssh command 'plink -ssh -agent -l stdedos -T 192.168.1.205 \'sh -c \'"\'"\'if command -v "xpra" > /dev/null 2>&1; then xpra _proxy_shadow_start ":1" "--opengl=no" "--title=@title@ on @@/@server-display@" "--ssh=plink -ssh -agent" "--microphone=off" "--speaker=off" "--webcam=no" "--headerbar=off" "--min-speed=70" "--pulseaudio=no" "--modal-windows=no" "--encodings=h264,vp9,vp8,mpeg4,mpeg4+mp4,h264+mp4,vp8+webm,vp9+webm,png,png/P,png/L,avif,rgb,rgb24,rgb32,jpeg,jpega,h265,av1,scroll,grayscale,stream" "--key-shortcut=Control+Menu:toggle_keyboard_grab" "--key-shortcut=Shift+Menu:toggle_pointer_grab" "--key-shortcut=Shift+F11:toggle_fullscreen" "--key-shortcut=#+F1:show_menu" "--key-shortcut=#+F2:show_start_new_command" "--key-shortcut=#+F3:show_bug_report" "--key-shortcut=#+F4:quit" "--key-shortcut=#+F5:show_window_info" "--key-shortcut=#+F6:show_shortcuts" "--key-shortcut=#+F7:show_docs" "--key-shortcut=#+F8:toggle_keyboard_grab" "--key-shortcut=#+F9:toggle_pointer_grab" "--key-shortcut=#+F10:magic_key" "--key-shortcut=#+F11:show_session_info" "--key-shortcut=#+F12:toggle_debug" "--key-shortcut=#+plus:scaleup" "--key-shortcut=#+minus:scaledown" "--key-shortcut=#+underscore:scaledown" "--key-shortcut=#+KP_Add:scaleup" "--key-shortcut=#+KP_Subtract:scaledown" "--key-shortcut=#+KP_Multiply:scalereset" "--key-shortcut=#+bar:scalereset" "--key-shortcut=#+question:scalingoff" "--key-shortcut=Shift+F2:toggle_pointer_grab";elif [ -x $XDG_RUNTIME_DIR/xpra/run-xpra ]; then $XDG_RUNTIME_DIR/xpra/run-xpra _proxy_shadow_start ":1" "--opengl=no" "--title=@title@ on @@/@server-display@" "--ssh=plink -ssh -agent" "--microphone=off" "--speaker=off" "--webcam=no" "--headerbar=off" "--min-speed=70" "--pulseaudio=no" "--modal-windows=no" "--encodings=h264,vp9,vp8,mpeg4,mpeg4+mp4,h264+mp4,vp8+webm,vp9+webm,png,png/P,png/L,avif,rgb,rgb24,rgb32,jpeg,jpega,h265,av1,scroll,grayscale,stream" "--key-shortcut=Control+Menu:toggle_keyboard_grab" "--key-shortcut=Shift+Menu:toggle_pointer_grab" "--key-shortcut=Shift+F11:toggle_fullscreen" "--key-shortcut=#+F1:show_menu" "--key-shortcut=#+F2:show_start_new_command" "--key-shortcut=#+F3:show_bug_report" "--key-shortcut=#+F4:quit" "--key-shortcut=#+F5:show_window_info" "--key-shortcut=#+F6:show_shortcuts" "--key-shortcut=#+F7:show_docs" "--key-shortcut=#+F8:toggle_keyboard_grab" "--key-shortcut=#+F9:toggle_pointer_grab" "--key-shortcut=#+F10:magic_key" "--key-shortcut=#+F11:show_session_info" "--key-shortcut=#+F12:toggle_debug" "--key-shortcut=#+plus:scaleup" "--key-shortcut=#+minus:scaledown" "--key-shortcut=#+underscore:scaledown" "--key-shortcut=#+KP_Add:scaleup" "--key-shortcut=#+KP_Subtract:scaledown" "--key-shortcut=#+KP_Multiply:scalereset" "--key-shortcut=#+bar:scalereset" "--key-shortcut=#+question:scalingoff" "--key-shortcut=Shift+F2:toggle_pointer_grab";elif [ -x /usr/local/bin/xpra ]; then /usr/local/bin/xpra _proxy_shadow_start ":1" "--opengl=no" "--title=@title@ on @@/@server-display@" "--ssh=plink -ssh -agent" "--microphone=off" "--speaker=off" "--webcam=no" "--headerbar=off" "--min-speed=70" "--pulseaudio=no" "--modal-windows=no" "--encodings=h264,vp9,vp8,mpeg4,mpeg4+mp4,h264+mp4,vp8+webm,vp9+webm,png,png/P,png/L,avif,rgb,rgb24,rgb32,jpeg,jpega,h265,av1,scroll,grayscale,stream" "--key-shortcut=Control+Menu:toggle_keyboard_grab" "--key-shortcut=Shift+Menu:toggle_pointer_grab" "--key-shortcut=Shift+F11:toggle_fullscreen" "--key-shortcut=#+F1:show_menu" "--key-shortcut=#+F2:show_start_new_command" "--key-shortcut=#+F3:show_bug_report" "--key-shortcut=#+F4:quit" "--key-shortcut=#+F5:show_window_info" "--key-shortcut=#+F6:show_shortcuts" "--key-shortcut=#+F7:show_docs" "--key-shortcut=#+F8:toggle_keyboard_grab" "--key-shortcut=#+F9:toggle_pointer_grab" "--key-shortcut=#+F10:magic_key" "--key-shortcut=#+F11:show_session_info" "--key-shortcut=#+F12:toggle_debug" "--key-shortcut=#+plus:scaleup" "--key-shortcut=#+minus:scaledown" "--key-shortcut=#+underscore:scaledown" "--key-shortcut=#+KP_Add:scaleup" "--key-shortcut=#+KP_Subtract:scaledown" "--key-shortcut=#+KP_Multiply:scalereset" "--key-shortcut=#+bar:scalereset" "--key-shortcut=#+question:scalingoff" "--key-shortcut=Shift+F2:toggle_pointer_grab";elif [ -x ~/.xpra/run-xpra ]; then ~/.xpra/run-xpra _proxy_shadow_start ":1" "--opengl=no" "--title=@title@ on @@/@server-display@" "--ssh=plink -ssh -agent" "--microphone=off" "--speaker=off" "--webcam=no" "--headerbar=off" "--min-speed=70" "--pulseaudio=no" "--modal-windows=no" "--encodings=h264,vp9,vp8,mpeg4,mpeg4+mp4,h264+mp4,vp8+webm,vp9+webm,png,png/P,png/L,avif,rgb,rgb24,rgb32,jpeg,jpega,h265,av1,scroll,grayscale,stream" "--key-shortcut=Control+Menu:toggle_keyboard_grab" "--key-shortcut=Shift+Menu:toggle_pointer_grab" "--key-shortcut=Shift+F11:toggle_fullscreen" "--key-shortcut=#+F1:show_menu" "--key-shortcut=#+F2:show_start_new_command" "--key-shortcut=#+F3:show_bug_report" "--key-shortcut=#+F4:quit" "--key-shortcut=#+F5:show_window_info" "--key-shortcut=#+F6:show_shortcuts" "--key-shortcut=#+F7:show_docs" "--key-shortcut=#+F8:toggle_keyboard_grab" "--key-shortcut=#+F9:toggle_pointer_grab" "--key-shortcut=#+F10:magic_key" "--key-shortcut=#+F11:show_session_info" "--key-shortcut=#+F12:toggle_debug" "--key-shortcut=#+plus:scaleup" "--key-shortcut=#+minus:scaledown" "--key-shortcut=#+underscore:scaledown" "--key-shortcut=#+KP_Add:scaleup" "--key-shortcut=#+KP_Subtract:scaledown" "--key-shortcut=#+KP_Multiply:scalereset" "--key-shortcut=#+bar:scalereset" "--key-shortcut=#+question:scalingoff" "--key-shortcut=Shift+F2:toggle_pointer_grab";elif [ -x Xpra_cmd.exe ]; then Xpra_cmd.exe _proxy_shadow_start ":1" "--opengl=no" "--title=@title@ on @@/@server-display@" "--ssh=plink -ssh -agent" "--microphone=off" "--speaker=off" "--webcam=no" "--headerbar=off" "--min-speed=70" "--pulseaudio=no" "--modal-windows=no" "--encodings=h264,vp9,vp8,mpeg4,mpeg4+mp4,h264+mp4,vp8+webm,vp9+webm,png,png/P,png/L,avif,rgb,rgb24,rgb32,jpeg,jpega,h265,av1,scroll,grayscale,stream" "--key-shortcut=Control+Menu:toggle_keyboard_grab" "--key-shortcut=Shift+Menu:toggle_pointer_grab" "--key-shortcut=Shift+F11:toggle_fullscreen" "--key-shortcut=#+F1:show_menu" "--key-shortcut=#+F2:show_start_new_command" "--key-shortcut=#+F3:show_bug_report" "--key-shortcut=#+F4:quit" "--key-shortcut=#+F5:show_window_info" "--key-shortcut=#+F6:show_shortcuts" "--key-shortcut=#+F7:show_docs" "--key-shortcut=#+F8:toggle_keyboard_grab" "--key-shortcut=#+F9:toggle_pointer_grab" "--key-shortcut=#+F10:magic_key" "--key-shortcut=#+F11:show_session_info" "--key-shortcut=#+F12:toggle_debug" "--key-shortcut=#+plus:scaleup" "--key-shortcut=#+minus:scaledown" "--key-shortcut=#+underscore:scaledown" "--key-shortcut=#+KP_Add:scaleup" "--key-shortcut=#+KP_Subtract:scaledown" "--key-shortcut=#+KP_Multiply:scalereset" "--key-shortcut=#+bar:scalereset" "--key-shortcut=#+question:scalingoff" "--key-shortcut=Shift+F2:toggle_pointer_grab";else echo "no run-xpra command found"; exit 1; fi\'"\'"\'\'': [WinError 2] Δεν είναι δυνατή η εύρεση του καθορισμένου αρχείου από το σύστημα

"was: Xpra-Light-x86_64_6.1-r35715\xpra_cmd" shadow ssh://user@ip/1 --ssh="plink -ssh -agent" --modal-windows=no --headerbar=off --title="@title@ on @@/@server-display@" --encodings=-webp --microphone=off --speaker=off --webcam=no --pulseaudio=no --opengl=no --min-speed=70 --key-shortcut=Shift+F2:toggle_pointer_grab

Press any key to continue . . .
```

**Comment by stdedos** [comment]
Seems to be a Light-issue

```python
"Xpra-x86_64_6.1-r35715\xpra_cmd" shadow ssh://user@ip/1 --ssh="plink -ssh -agent" --modal-windows=no --headerbar=off --title="@title@ on @@/@server-display@" --encodings=-webp --microphone=off --speaker=off --webcam=no --pulseaudio=no --opengl=no --min-speed=70 --key-shortcut=Shift+F2:toggle_pointer_grab

XPRA_EXECUTABLE=Xpra-x86_64_6.1-r35715


2024-05-04 13:04:27,221 Xpra GTK3 client version 6.1-r35715 (g1ec038bf9a) beta
2024-05-04 13:04:27,525  running on Microsoft Windows Microsoft Windows 10 Enterprise
2024-05-04 13:04:27,525  cpython 3.11
2024-05-04 13:04:28,406 GStreamer version 1.24.3
2024-05-04 13:04:28,431 created named pipe 'Xpra\34072'
2024-05-04 13:04:28,705 Error setting up the pipeline:
2024-05-04 13:04:28,705  gst_parse_error: could not link decoder to sink (3)
2024-05-04 13:04:28,705  GStreamer pipeline for:
2024-05-04 13:04:28,705   appsrc name=src emit-signals=1 block=0 is-live=1 do-timestamp=1 stream-type=0 format=2 caps=video/x-h264,width=128,height=128,profile=(string)main,stream-format=(string)byte-stream,alignment=(string)au ! \
2024-05-04 13:04:28,706   d3d11h264dec name=decoder ! \
2024-05-04 13:04:28,706   appsink name=sink emit-signals=1 max-buffers=10 drop=False sync=False async=True qos=False caps=video/x-raw,width=128,height=128,format=(string)I420
2024-05-04 13:04:28,706 Error creating context h264 128x128 YUV420P
2024-05-04 13:04:28,706 gstreamer: h264 decoding failed: failed to setup gstreamer pipeline
2024-05-04 13:04:28,718 keyboard layout 'United States - English' : 'us' (0x409)
2024-05-04 13:04:28,852  keyboard settings: layout=us
2024-05-04 13:04:28,854  desktop size is 4480x1516:
2024-05-04 13:04:28,854   Default (1185x401 mm - DPI: 96x96) workarea: 4480x1476
2024-05-04 13:04:28,854     Samsung C32JG5x  2560x1440 at    0x0    (697x392 mm - DPI: 93x93) workarea: 2560x1400
2024-05-04 13:04:28,854     LG IPS234        1920x1080 at 2560x436  (510x290 mm - DPI: 96x95) workarea: 1920x1040 at 2560x436
Exception in thread ssh-stderr-reader:
Traceback (most recent call last):
  File "C:/msys64/mingw64/lib/python3.11/threading.py", line 1045, in _bootstrap_inner
  File "C:/msys64/mingw64/lib/python3.11/threading.py", line 982, in run
  File "E:/xpra/xpra/net/ssh/exec_client.py", line 223, in stderr_reader
TypeError: a bytes-like object is required, not 'str'
2024-05-04 13:04:32,903 enabled remote logging
2024-05-04 13:04:32,905 Xpra shadow server version 5.0
2024-05-04 13:04:32,907  remote desktop size is 1920x1200
2024-05-04 13:04:32,907   :1.0 (508x318 mm - DPI: 96x96) workarea: 1848x1173 at   72x27
2024-05-04 13:04:32,908     LGD eDP-1        (336x210 mm - DPI: 145x145)
2024-05-04 13:04:33,103 running, 1 windows
```

(also note the `Exception in thread ssh-stderr-reader:` in the above code block)

**Comment by totaam** [comment]
> (also note the Exception in thread ssh-stderr-reader: in the above code block)

Thanks, fixed in d61a48e370ac46485621c930c3c790b9ef9b55d4 - I just broke it yesterday!

> Seems to be a Light-issue

Of course it is: the "Light" builds don't include `plink.exe`!

I won't divulge how much time I wasted looking in the wrong place as it's too embarrassing, a1f70d484c1b97e7c7532cf42479f1b58d5eef7d will make this clearer. Try `6.1-r35724` or later.
Closing, optimistically.

**Comment by stdedos** [comment]
Verified as "fixed" in `Xpra-Light-x86_64_6.1-r35724`

```python
2024-05-04 21:07:46,907 keyboard layout 'United States - English' : 'us' (0x409)
2024-05-04 21:07:46,926 Warning: ssh command 'plink' not found!
2024-05-04 21:07:46,926  trying to continue anyway..
```
